cli/command/swarm/opts.go

```diff
@@ -19,6 +19,7 @@ const (
 	flagDispatcherHeartbeat = "dispatcher-heartbeat"
 	flagListenAddr          = "listen-addr"
 	flagAdvertiseAddr       = "advertise-addr"
+	flagDataPathAddr        = "datapath-addr"
```

Comment:
Wondering if this should be `data-path-addr`. Not sure either way but wanted to bring it up for discussion.